src/src/Controller/UserController.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\User;
+use App\Repository\UserRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\PasswordHasher\Hasher\UserPasswordHasherInterface;
+
+class UserController extends AbstractController
+{
+
+    #[Route('/user', name: 'app_user_index')]
+    public function index(UserRepository $repository): Response
+    {
+        $users = $repository->findAll();
+        // dd($users);
+        return $this->render('user/index.html.twig', [
+            'users' => $users,
+        ]);
+    }
+
+    #[Route('/user/new', name: 'app_user_new')]
+    public function addUser(): Response
+    {
+        return $this->render('user/user_new.html.twig');
+    }
+
+    #[Route('/user/create', name: 'app_user_create_api', methods: 'POST')]
```

Comment:
Why `/user/create` and not just `/user/new` why POST method ?

Reply:
Post method is required haha, yeah create or new doesn’t change smtg. theres a new fand create is not the same